templates/registration/registration.html

```diff
@@ -0,0 +1,115 @@
+{% load static %}
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>AcadeMe</title>
+    <link rel="stylesheet" href="{%  static  'css/landingStyle.css'  %}" />
+    <link
+      href="//maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+      rel="stylesheet"
+      id="bootstrap-css"
+    />
+    <script src="//maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+  </head>
+
+  <body>
+    <link
+      href="//maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+      rel="stylesheet"
+      id="bootstrap-css"
+    />
+    <script src="//maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"></script>
+    <script src="//cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+    <!------ Include the above in your HEAD tag ---------->
+
+    <div class="wrapper fadeInDown">
+      <div id="formContent">
+        <!-- Tabs Titles -->
+
+        <!-- Icon -->
+        <div class="fadeIn first">
+          <div class="col-md-5">
+            <img
+              src="{%  static   'img/logo.jpg' %}"
+              style="width: 270%; height: 260%"
+              alt="home"
+            />
+          </div>
+        </div>
+
+        <!-- Login Form -->
+        <form>
+          <input
+            type="text"
+            id="login"
+            class="fadeIn second"
+            name="login"
+            placeholder="Choose Username"
+          />
+          <input
+            type="text"
+            id="password"
+            class="fadeIn third"
+            name="login"
+            placeholder="E-Mail"
+          />
+          <input
+            type="text"
+            id="password"
+            class="fadeIn fourth"
+            name="login"
+            placeholder="Choose Password"
+          />
+          <input
+            type="text"
+            id="password"
+            class="fadeIn fourth"
+            name="login"
+            placeholder="Confirm Password"
+          />
+          <div class="justify-content-center">
+            <select
+              class="fadeIn fourth"
+              aria-label="choose-university"
+              id="selectContent"
+            >
+              <option selected>Choose University</option>
```

Comment:
once you connect a backend to it, replace the manual option section.

Reply:
I indeed think it should be queried.

Reply:
I am not sure I understand what to do, you want me to fetch this data from the test migrations?
If so how should I approach it?

Reply:
Please refer UI talk see the post_list example.
With a for loop in the django template, only thing you have to dod is query universities from DB and parse it as parameter when you render the page,

then the template iterate the parameter with for tag in django template